backend/db_funcs/config.py

```diff
@@ -0,0 +1,54 @@
+import json
+from configparser import ConfigParser
+import psycopg2
+import psycopg2.extras
+from psycopg2._psycopg import connection
+
+
+class DB_conn:
+    conn = ""
+    INI_FILENAME = 'database.ini'
+    SETTINGS_FILENAME = 'settings.json'
+
+    def __init__(self):
+        self.chooseDB()
+
+    def chooseDB(self):
+        with open(self.SETTINGS_FILENAME, 'r') as dbFile:
+            data = dbFile.read()
+
+        obj = json.loads(data)
+
+        if obj['database'] == 'dev':
+            DB_conn.create_connection(self, self.INI_FILENAME, 'dev')
+
+        elif obj['database'] == 'prod':
+            DB_conn.create_connection(self, self.INI_FILENAME, 'prod')
+
+        elif obj['database'] == 'local':
+            DB_conn.create_connection(self, self.INI_FILENAME, 'local')
```

Comment:
Nešlo by zjednodušit?

`DB_conn.create_connection(self, self.INI_FILENAME, obj['database'])`

a kontrolovat, jestli se to povedlo